### ISSUE TITLE
Hosts being allocated 2 IPV4 IPAM affine blocks at start-of-day

### ISSUE THREAD

**Issue by fasaxc** [projectcalico/calico-containers#1378]
Spotted by @MikeSpreitzer under #1377 (runing Calico 1.6).  I checked my test cluster and it also seems to have 2 blocks per host even though I've never had more than about 10 pods running.

**Comment by fasaxc** [comment]
Example from my k8s test cluster; routes show that each host has two blocks (I also checked that same was true in etcd):
```
192.168.120.128/26 via 10.240.0.30 dev tunl0  proto bird onlink 
192.168.7.192/26 via 10.240.0.30 dev tunl0  proto bird onlink 
192.168.189.128/26 via 10.240.0.31 dev tunl0  proto bird onlink 
192.168.60.128/26 via 10.240.0.31 dev tunl0  proto bird onlink 
blackhole 192.168.155.128/26  proto bird 
blackhole 192.168.255.64/26  proto bird 
```

**Comment by MikeSpreitzer** [comment]
Thanks, I've been meaning to raise this as an issue of its own.  I have been seeing it for a while.  At least in 1.5 and 1.6.

**Comment by caseydavenport** [comment]
@robbrockbank we just spoke about this today in the context of v2.0.0.

See this libcalico issue: https://github.com/projectcalico/libcalico-go/issues/294

The intention is for @gunjan5 to fix this in time for v2.0.0.

**Comment by caseydavenport** [comment]
Closing this in favor of the libcalico issue since that's where the offending code is.

See projectcalico/libcalico-go#294
